alf/trainers/policy_trainer.py

```diff
@@ -698,3 +760,108 @@ def play(root_dir,
     if recorder:
         recorder.close()
     env.reset()
+
+
+def _record_future_info(recorder,
```

Comment:
This function doesn't have to be separate from the existing. All the new stuff can be put into VideoRecorder class's capture_frame() (by letting it accept a policy_step) without bloating policy_trainer.py code

Reply:
Refactored and moved into VideoRecoder.